packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -259,7 +259,7 @@ export function usePress(props: PressHookProps): PressResult {
 
           // If triggered from a screen reader or by using element.click(),
           // trigger as if it were a keyboard click.
-          if (!state.ignoreClickAfterPress && !state.ignoreEmulatedMouseEvents && isVirtualClick(e.nativeEvent)) {
+          if (!state.ignoreClickAfterPress && !state.ignoreEmulatedMouseEvents && (state.pointerType === 'virtual' || isVirtualClick(e.nativeEvent))) {
```

Comment:
This change is specifically to make this PR work with the changes made in https://github.com/adobe/react-spectrum/commit/649ac075260190e88ee6d9de7e32f9646d69d4ac#diff-e93ec4d995a5d193a3779d4879091f2fd9e8199ceffc3d917fa823cfd2622737 (PR: https://github.com/adobe/react-spectrum/pull/2510/)

The above commit defers triggering the press events to `onClick` when `onPointerDown`'s event is a virtual pointer event. The problem that then occurs is that `isVirtualClick` thinks the click event from Android TalkBack double tap is NOT a virtual click because the event has `detail: 1` and `pointerType: 'mouse'`. I would love to make `isVirtualClick` properly detect the Android TalkBack double tap but doesn't seem to be a great way to distinguish the TalkBack click event from a browser click event. Here is a diff (on the left is a browser event, on the right is the Android TalkBack event): https://www.diffchecker.com/ccP88a4H. I could possible use the `pointerType: mouse` + `buttons: 1` + `isPrimary: true` + `type: click` as a way to distinguish the Android TalkBack event from browser clicks and from other pointer events but feels kinda iffy

The above change works around this issue by using `state.pointerType` over the `isVirtualClick` check if it exists since  usePress `onPointerDown` will properly detect the Android TalkBack double as a virtual pointer event and set `state.pointerType` accordingly. `onPointerDown` seems to always fire before `onClick` which seems to be backed up by what is described in the w3 pointer spec.

The Android TalkBack event structure seems to be a bug though as suggested by https://w3c.github.io/pointerevents/#dfn-click-auxclick-and-contextmenu-events which states that "For these events (read as click), all PointerEvent specific attributes (defined in this spec) other than pointerId and pointerType will have their default values". Will file a bug

defaults: https://w3c.github.io/pointerevents/#pointerevent-interface

Reply:
https://issuetracker.google.com/issues/206039945

Reply:
Slightly concerning given that we use `isVirtualClick` in some other places as well... It's weird that Chrome appears to have started emitting `PointerEvent` values for the `click` event rather than `MouseEvent`. Seems like no other browser does this. Looks like it's possibly broken even in desktop VoiceOver in Chrome too (`e.detail === 1`). That might cause issues if the talkback check doesn't also apply to VO?

Reply:
Looks like https://groups.google.com/a/chromium.org/g/blink-dev/c/bta50W_Hg24

Reply:
Yeah, the stuff in that link looks like it matches the [w3c specs for click](https://w3c.github.io/pointerevents/#the-click-auxclick-and-contextmenu-events), perhaps an attempt to move towards a new standard

Reply:
Hm, looks like `buttons: 1` and `isPrimary: true` is the only thing that distinguishes desktop Chrome click vs desktop Chrome VO click for now...